compiler/fir/providers/src/org/jetbrains/kotlin/fir/scopes/impl/FirFakeOverrideGenerator.kt

```diff
@@ -476,6 +477,10 @@ object FirFakeOverrideGenerator {
             annotations += baseField.annotations
             attributes = baseField.attributes.copy()
             dispatchReceiverType = baseField.dispatchReceiverType
+            deprecationsProvider = baseField.deprecationsProvider
+            if (withInitializer) {
+                initializer = baseField.initializer
```

Comment:
Why do you need this? Looks like the codegen test you have added works without it.

Reply:
There should be a bytecode text test somewhere that checks that a `static final` field of primitive type in an interface is correctly inlined into use sites (it has a ConstantValue JVM attribute). I don't remember which one that is. Without this, reading that field through a subclass will cause it to not be inlined.

Reply:
Ok, so without this change this bytecode test will become broken. Thanks.